rootfs/api/models/app.py

```diff
@@ -171,10 +171,27 @@ def create(self, *args, **kwargs):
             )
 
         # create required minimum resources in k8s for the application
+        namespace = self.id
+        service = self.id
         try:
-            self._scheduler.create(self.id)
+            self.log('creating Namespace {} and services'.format(namespace), level=logging.DEBUG)
+            # Create essential resources
+            try:
+                self._scheduler._get_namespace(namespace)
```

Comment:
technically speaking we shouldn't be using private functions (functions with `_` before the name) outside of the package; only public functions. If we want to migrate these over, we should make `_get_namespace`, `_create_namespace`, etc. as public functions.